google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/BuiltinMetricsConstants.java

```diff
@@ -0,0 +1,165 @@
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.data.v2.stub.metrics;
+
+import com.google.api.core.InternalApi;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.sdk.metrics.Aggregation;
+import io.opentelemetry.sdk.metrics.InstrumentSelector;
+import io.opentelemetry.sdk.metrics.InstrumentType;
+import io.opentelemetry.sdk.metrics.View;
+import java.util.Map;
+import java.util.Set;
+
+/** Defining Bigtable builit-in metrics scope, attributes, metric names and views. */
+@InternalApi
+public class BuiltinMetricsConstants {
+
+  public static final AttributeKey<String> PROJECT_ID = AttributeKey.stringKey("project_id");
+  public static final AttributeKey<String> INSTANCE_ID = AttributeKey.stringKey("instance");
+  public static final AttributeKey<String> TABLE_ID = AttributeKey.stringKey("table");
+  public static final AttributeKey<String> CLUSTER_ID = AttributeKey.stringKey("cluster");
+  public static final AttributeKey<String> ZONE_ID = AttributeKey.stringKey("zone");
+  static final AttributeKey<String> CLIENT_UID = AttributeKey.stringKey("client_uid");
+
+  public static final AttributeKey<String> APP_PROFILE = AttributeKey.stringKey("app_profile");
+  public static final AttributeKey<Boolean> STREAMING = AttributeKey.booleanKey("streaming");
+  static final AttributeKey<String> METHOD = AttributeKey.stringKey("method");
+  static final AttributeKey<String> STATUS = AttributeKey.stringKey("status");
+  static final AttributeKey<String> CLIENT_NAME = AttributeKey.stringKey("client_name");
+
+  public static final String OPERATION_LATENCIES_NAME = "operation_latencies";
+  static final String ATTEMPT_LATENCIES_NAME = "attempt_latencies";
+  static final String RETRY_COUNT_NAME = "retry_count";
+  static final String CONNECTIVITY_ERROR_COUNT_NAME = "connectivity_error_count";
+  static final String SERVER_LATENCIES_NAME = "server_latencies";
+  static final String FIRST_RESPONSE_LATENCIES_NAME = "first_response_latencies";
+  static final String APPLICATION_BLOCKING_LATENCIES_NAME = "application_latencies";
+  static final String CLIENT_BLOCKING_LATENCIES_NAME = "throttling_latencies";
+
+  private static final Aggregation AGGREGATION_WITH_MILLIS_HISTOGRAM =
+      Aggregation.explicitBucketHistogram(
+          ImmutableList.of(
+              0.0, 0.01, 0.05, 0.1, 0.3, 0.6, 0.8, 1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 8.0, 10.0, 13.0,
```

Comment:
Please add a note where these buckets came from. I assume they are the buckets used for the server side metrics?